distributions/Jamulus.1

```diff
@@ -29,7 +29,7 @@
 .Op Fl P | Fl \-delaypan
 .Op Fl p | Fl \-port Ar number
 .Op Fl Q | Fl \-qos Ar value
-.Op Fl R | Fl \-recording Ar directory
+.Op Fl R | Fl \-recording Ar Directory
```

Comment:
How do these appear?  Single dash for the short option, clearly a double dash (not a long dash) for the long options?

Reply:
I suppose this means I'm going to have to work out how to compile a man page locally... Sigh.

Reply:
No need, `man` will do it for you. Just do `man distributions/Jamulus.1` to view it.

Reply:
`Fl x` renders as `-x` (Fl means flag). GNU --long-options must be `Fl \-long\-option` (manpages are a BSD thing, GNU uses texinfo instead, so there’s no provision for GNU long options in manpages).

@softins `man -l distributions/Jamulus.1` (`-l` = local) might be needed.

Reply:
> @softins `man -l distributions/Jamulus.1` (`-l` = local) might be needed.

It wasn't needed on my Pi running Raspbian 11.

In fact looking at `man man`, I see that it automatically tries again with an implicit `-l`, if it can't match the supplied argument with an installed man page, such as when giving a pathname instead of a simple name and section.